README.md

```diff
@@ -26,6 +32,24 @@ over (barlow (key :: _ "zodiac.virgo.alpha")) toUpper sky
 -- doesn't compile
 ```
 
+### Deep sky 🌌
+
+Now you can also zoom into Maybes! 
+
+```purescript 
+sky =
+  { zodiac:
+      Just
+        { virgo:
+            Just
+              { alpha: Just "Spica"
+              }
+        }
+  }
+
+preview (barlow (key :: _ "zodiac?.virgo?.alpha?")) sky
```

Comment:
maybe have one layer not be a `Maybe` to make it clearer that that also works (that does work, right?)